### PR TITLE
R4R: Refactor bank query using Querier

### PR DESCRIPTION
**Pull request by chengwenxi**
Ref: https://github.com/irisnet/irishub/issues/1374

### CODE REVIEW

**Comment by HaoyangLiu** [comment]
The querier interface can't provide merkle proof. Is this acceptable?

**Comment by haifengxi** [comment]
> The querier interface can't provide merkle proof. Is this acceptable?
For queries, we'd expect clients to always connect to some node that they trust, so I guess that's an ok tradeoff between performance and trustworthiness. @HaoyangLiu

**Comment by HaoyangLiu** [comment]
But once this PR is merged, there is no other command which can query account info and provide merkle proof.

**Comment by haifengxi** [comment]
> But once this PR is merged, there is no other command which can query account info and provide merkle proof.

Got your point.  Account query LCD API is the only one left that can provide a merkle proof, which is true for  Cosmos SDK as well.  The accuracy and trustworthiness of **account info** has much more weight than other on-chain artifacts such as distribution rewards for example.

We'll leave this specific LCD API intact then.  @chengwenxi

File: modules/auth/querier.go [hunk 1]
```diff
@@ -55,3 +58,21 @@ func queryAccount(ctx sdk.Context, req abci.RequestQuery, keeper AccountKeeper)
...
+		return nil, sdk.ErrInternal(sdk.AppendMsgToErr("could not marshal result to JSON", err.Error()))
+	}
+
+	return bz, nil
+}
+
+type TokenStats struct {
+	LoosenToken sdk.Coins `json:"loosen_token"`
+	BurnedToken sdk.Coins `json:"burned_token"`
+}
```
Comment (HaoyangLiu) [inline modules/auth/querier.go]: How about adding `BondedToken`?

**Pull request merged**